examples/duel/.gitignore

```diff
@@ -1 +1,3 @@
 target/
+node_modules/
```

Comment:
You'll have to build up a proper gitignore (*[example](https://github.com/tastejs/todomvc/blob/master/examples/dojo/.gitignore)*) and commit the dependencies needed to run the app.